src/browser/atom-application.coffee

```diff
@@ -167,7 +167,7 @@ class AtomApplication
     @on 'application:run-benchmarks', -> @runBenchmarks()
     @on 'application:quit', -> app.quit()
     @on 'application:new-window', -> @openPath(_.extend(windowDimensions: @focusedWindow()?.getDimensions(), getLoadSettings()))
-    @on 'application:new-file', -> (@focusedWindow() ? this).openPath()
+    @on 'application:new-file', -> @openPath()
```

Comment:
Late response: On OSX, it is possible that there is no focused window (eg. when all windows are closed), but the application might still be active and receive keyboard events. Native applications still allow users to open new files via keyboard shortcuts, which isn't supported by Atom.

If there is no focused window, then `focusedWindow` returns undefined and the command is ignored. Removing this check seemed like a quick fix. Having a second look, the `newWindow` option of `openPath` should instead probably be set to `Boolean(@focusedWindow())`.

Any thoughts?